src/mapeo-project.js

```diff
@@ -717,9 +724,19 @@ export class MapeoProject extends TypedEmitter {
    *  @returns {Promise<Error[]>}
    */
   async importConfig({ configPath }) {
+    if (this.#loadingConfig) return /** @type Error[] */ []
```

Comment:
If I call `importConfig({ configPath: 'path 1' })` and then `importConfig({ configPath: 'path 2' })`, the second one will be silently dropped. Is that what we want?

IMO, for simplicity, this should throw an error instead.

```suggestion
    assert(!this.#loadingConfig, 'Cannot run multiple config imports at the same time')
```